exonum-java-binding/core/src/main/java/com/exonum/binding/core/service/Service.java

```diff
@@ -53,6 +53,26 @@ default void initialize(Fork fork, Configuration configuration) {
     // No configuration
   }
 
+  /**
+   * Resumes the previously stopped service instance. This method is called when
+   * a stopped service instance is restarted.
+   *
+   * <p>This method may perform any changes to the database. For example, update some service
+   * parameters, deprecate old entries etc.
+   *
+   * <p>Also, note that performing any bulk operations or data migration
+   * <em>is not recommended</em> here.
+   * <!--TODO: Add a link to the migration procedure -->
+   *
+   * @param fork a database fork to apply changes to. Not valid after this method returns
+   * @param configuration the service configuration parameters
```

Comment:
As we've discussed, those are hardly _configuration parameters_, just generic 'arguments'.

Reply:
`byte[]` since is it is not configuration. We shall not use Configuration for this operation — it is for configuration parameters.

It could be made one, it is made in this way probably to give more flexibility in terms of implementation (e.g., if we went with also properties).